### DISCUSSION TITLE
How to make tags appear?

### DISCUSSION THREAD

**Discussion by Fierce-Ninja** [HEmile/juggl#94] [Q&A]
Hi there, I recently installed Juggl in Obsidian. For the life of me, I cannot figure out how to make the tags appear in the graph.  I was looking for some kind of universal setting in Juggl that would turn on tags display along with notes.  But didn't find anything. I am new.  So guessing I am missing something obvious here.  Could someone please help?

**Comment by HEmile** [comment]
You're not missing anything! In Juggl, tags are not part of the graph, but a feature of the node that you can use for styling.

**Reply by Fierce-Ninja** [reply]
@HEmile thank you.  I am trying to follow you here.  Are you saying that tags don't exist as separate nodes as they do in the graph view of Obsidian?  Or are you saying that I can make tags appear if I did something with styling?  Sorry noob here to Juggl, so haven't yet done much with styling.

**Reply by HEmile** [reply]
The first one! You cannot do it with styling.
